core/src/main/java/org/apache/accumulo/core/file/rfile/bcfile/BoundedRangeFileInputStream.java

```diff
@@ -95,18 +92,7 @@ public int read(final byte[] b, final int off, int len) throws IOException {
         throw new IOException("Stream closed");
       }
       in.seek(pos);
-      try {
-        ret = AccessController.doPrivileged(new PrivilegedExceptionAction<Integer>() {
```

Comment:
I don't remember any discussion at the hackathon... I'm pretty sure it just happened to be where I was when I committed it. I don't think you missed anything.